### DISCUSSION TITLE
Enable Leveraging of Spack Testing by Test Frameworks

### DISCUSSION THREAD

**Discussion by tldahlgren** [spack/spack#38454] [Ideas]
This discussion is intended to allow those interested in leveraging Spack testing features by test frameworks (e.g., PCVS) a place to exchange ideas.  Areas of interest can include leveraging stand-alone tests implemented in Spack packages to conduct tests on the software built with different options (or variants) and to perform testing at scale.

There have been several proposals related to adding capabilities to Spack (e.g., options to existing commands) to provide information that can be used by the frameworks.  These include:

- https://github.com/spack/spack/issues/37025  (see #38456)

**Comment by tldahlgren** [comment]
@sameershende @wspear @tgamblin @becker33 @besnardjb @gweodoo I am not sure how much information to add to this discussion at this point but wanted to ping you all so you have the link.

**Comment by wspear** [comment]
I think there is some intersection with this issue: https://github.com/spack/spack/pull/22703 in that if we had an MPI runner, or some kind of standardized procedure for launching tests, that would provide a chunk of what we need here.
